account_analytic_line_product_root_category/README.rst

```diff
@@ -0,0 +1,26 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Product root category on analytic lines
+=======================================
+Adds product root category to analytic line and also adds filter and
```

Comment:
This module adds product's root category to analytic lines and it enables filtering and grouping